electron_app/DownloadManager.js

```diff
@@ -0,0 +1,55 @@
+const mtd = require('zeltice-mt-downloader');
+const path = require('path');
+const os = require('os');
+
+const handleFolder = require('./helpers/_handleDisk');
+
+let downElements = [];
+
+let downloadManager = (items, access_token) => {
+    let filePath = '';
```

Comment:
It looks like this code and `_handleDisk.js` uses four space indent, but other files use two space indent. I think we had decided on two space indent but can't recall for sure.

Reply:
Yes, I miss that in the .JS files.